Add kochv1.1 updated version of Alex Koch low-cost robot arm

s1lent4gnt opened the pull request:
This PR adds the updated Kochv1.1 low-cost robot arm model, developed by the HuggingFace LeRobot Team based on Alex Koch's design. Referencing #142.

s1lent4gnt commented:
Hi @kevinzakka,

Just wanted to check if you had a chance to review this PR. 
Let me know if any changes are needed. Thanks!

s1lent4gnt commented:
Hi @kevinzakka, just following up on this PR. 
Let me know if you need any modifications or if there's anything I can do to help. Thanks again!

kevinzakka commented on low_cost_robot_arm/low_cost_robot_arm.xml:
```diff
@@ -0,0 +1,136 @@
+  <keyframe>
+    <key name="home" qpos="0 0 0 0 0 0"/>
+  </keyframe>
+
```
Nit: Remove final newline between keyframe and mujoco tag.

kevinzakka commented on low_cost_robot_arm/low_cost_robot_arm.xml:
```diff
@@ -0,0 +1,136 @@
+  </default>
+
+  <asset>
+    <material name="black" rgba="0.1 0.1 0.1 1" specular="0.1" shininess="0.1"/>
```
Nit: I've found that specular and shininess aren't really noticeable in the Menagerie scenes so feel free to remove.

kevinzakka commented on low_cost_robot_arm/low_cost_robot_arm.xml:
```diff
@@ -0,0 +1,136 @@
+<mujoco model="low_cost_robot_arm">
+  <compiler angle="radian" meshdir="./assets/"/>
```
Can you just change this to `meshdir="assets"`?

kevinzakka commented on low_cost_robot_arm/low_cost_robot_arm.xml:
```diff
@@ -0,0 +1,136 @@
+<mujoco model="low_cost_robot_arm">
+  <compiler angle="radian" meshdir="./assets/"/>
+
+  <option cone="elliptic" impratio="10"/>
```
Please switch to integrator="implicitfast" when using explicit position actuators (aka kp and kv). Also remove joint damping.

kevinzakka commented on low_cost_robot_arm/low_cost_robot_arm.xml:
```diff
@@ -0,0 +1,136 @@
+  </asset>
+
+  <worldbody>
+    <body name="base_link" childclass="low_cost_robot_arm" quat="-0.707 0. 0. 0.707">
```
Is the quat on the base body necessary?

s1lent4gnt replied:
It’s not strictly necessary, I added the quaternion to align the arm’s forward axis with the X-axis. When I created the URDF, I didn’t pay attention to the axis conventions. Do you think it’s fine to keep it like this, or should I regenerate the URDF to align the axes properly for a cleaner setup?

kevinzakka commented on low_cost_robot_arm/low_cost_robot_arm.xml:
```diff
@@ -0,0 +1,136 @@
+      <geom mesh="base_link" class="visual" material="white"/>
+      <geom mesh="base_link_motor" class="visual" material="black"/>
+      <geom mesh="base_link_collision" class="collision"/>
+      
```
Can you remove all newlines between the bodies? In general, it would be great if you could follow our [XML guide](https://github.com/google-deepmind/mujoco_menagerie/blob/main/CONTRIBUTING.md#xml-style), in particular using the auto formatter.

kevinzakka commented on low_cost_robot_arm/low_cost_robot_arm.xml:
```diff
@@ -0,0 +1,136 @@
+  </actuator>
+
+  <keyframe>
+    <key name="home" qpos="0 0 0 0 0 0"/>
```
Don't forget `ctrl` in the keyframe since you are using position actuators.

kevinzakka reviewed (changes requested):
Thanks @s1lent4gnt. Can you 1) rebase and 2) add an entry in the top level README under arms? Cheers!

s1lent4gnt commented:
Hi @kevinzakka, Thanks for the feedback !

 I’ve rebased the branch, updated the README under "arms", and made the changes based on your comments. Everything should be good to go now. Let me know if there's anything else you'd like me to adjust. Cheers!
